pkg/users/account-create-dialog.js

```diff
@@ -118,6 +127,26 @@ function validate_real_name(real_name) {
         return _("The full name must not contain colons.");
 }
 
+function validate_uid(uid, accounts, min_id, max_id, change) {
+    if (!uid)
+        return _("No UID specified");
+
+    const uid_number = Number(uid);
+    if (!Number.isInteger(uid_number) || uid_number < 0)
+        return _("UID must be a positive integer.");
```

Comment:
This ends with a period, while other error messages don't. I'm not sure what our convention is, @garrett ?

Reply:
I believe if the error message is only one sentence, then there is no need for a period.